discovery-provider/alembic/env.py

```diff
@@ -96,6 +115,11 @@ def run_migrations_online():
         with context.begin_transaction():
             context.run_migrations()
 
+        try:
+            load_triggers(connection)
+        except Exception as e:
```

Comment:
It's definitely scary to make this non-fatal to me, however we also publish the current alembic version

https://discoveryprovider.audius.co/alembic_version

Perhaps we should put that in healthz for quick monitoring